src/PHPMailer.php

```diff
@@ -1853,12 +1855,12 @@ public function smtpConnect($options = null)
             }
             $prefix = '';
             $secure = $this->SMTPSecure;
-            $tls = ('tls' == $this->SMTPSecure);
-            if ('ssl' == $hostinfo[2] or ('' == $hostinfo[2] and 'ssl' == $this->SMTPSecure)) {
+            $tls = ('tls' === $this->SMTPSecure);
```

Comment:
I know many go on about using strict comparisons, but this is often done dogmatically rather than for any specific reason. In cases like this we *really* don't care, and it *really* doesn't matter whether it's actually a string or something that can be coerced into one. The value submitted is never actually used directly, only compared against, and such unnecessary strictness runs counter to usability, very much an instance of Postel's law.

Reply:
yeah, i picked only comparisons, that compare strings against strings as that is considered safe.

anyway, removing these two commits from this PR:
- Applying: strict in_array comparision
- Applying: strict string comparison